tgui/packages/tgui/interfaces/PreferencesMenu/CharacterPreferenceWindow.tsx

```diff
@@ -23,21 +23,25 @@ const CharacterProfiles = (props: {
   onClick: (index: number) => void,
   profiles: (string | null)[],
 }) => {
-  const { profiles } = props;
+  const { profiles, activeSlot, onClick } = props;
 
   return (
-    <Stack justify="center" wrap>
-      {profiles.map((profile, slot) => (
-        <Stack.Item key={slot}>
-          <Button
-            selected={slot === props.activeSlot}
-            onClick={() => {
-              props.onClick(slot);
-            }} fluid>{profile ?? "New Character"}
-          </Button>
-        </Stack.Item>
-      ))}
-    </Stack>
+    <Flex align="center" justify="center">
+      <Flex.Item width="25%">
+        <Dropdown
+          width="100%"
+          selected={activeSlot}
+          displayText={profiles[activeSlot]}
```

Comment:
`align="center"` and `selected` don't seem to be needed for this. LGTM otherwise.
```suggestion
    <Flex justify="center">
      <Flex.Item width="25%">
        <Dropdown
          width="100%"
          displayText={profiles[activeSlot]}
```